k8s/upgrades/0.8.0-0.8.1/cstor-pool-patch.tpl.json

```diff
@@ -0,0 +1,50 @@
+{
+  "metadata": {
+     "labels": {
+        "openebs.io/cas-template-name": "cstor-pool-create-default-0.8.1",
+        "openebs.io/version": "0.8.1"
+     }
+  },
+  "spec": {
+     "template": {
+        "spec": {
+           "containers": [
+              {
+                 "name": "cstor-pool",
+                 "image": "quay.io/openebs/cstor-pool:0.8.1",
+                 "env": [
+                   {
+                     "name": "OPENEBS_IO_CSTOR_ID",
+                     "value": "@csp_uuid"
```

Comment:
Why are we patching csp uuid.
I guess csp uuid will remain same always.

Reply:
Yes, It is remaining same I will remove this from patch.